### PR TITLE
DM-20382: Add description of RC2 ap_pipe repos

### PR DESCRIPTION
**Pull request by yalsayyad**

### CODE REVIEW

**Comment by yalsayyad** [comment]
Includes all and "best seeing" visit lists for tract 9813.

File: false_positives/ap_pipe_HSC_RC2.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+mkdir /datasets/hsc/repo/rerun/private/yusra/RC2/ap_pipe_PDR1_bestcase_noSkyCorr
+make_ppdb.py --config ppdb.isolation_level=READ_UNCOMMITTED ppdb.db_url="sqlite:////datasets/hsc/repo/rerun/private/yusra/RC2/ap_pipe_PDR1_bestcase_noSkyCorr/association.db
+```
+
+## Run ap_pipe:
+
+ Out of habit I direct to Hsin-Fang's latest monthly rerun `RC/w_2019_22/DM-19244`.  This will just redirect up the parent tree to the raws in `/datasets/hsc` when we ask for raws.
+
+```
+ap_pipe.py /datasets/hsc/repo --calib /datasets/hsc/repo/CALIB/ --template /datasets/hsc/repo/rerun/private/yusra/RC2/ap_pipe_templates_noSkyCorr --rerun RC/w_2019_22/DM-19244:private/yusra/RC2/ap_pipe_PDR1_bestcase_noSkyCorr --configfile ./configfile.py --no-versions --longlog --id visit=1228 ccd=0..8^10..103
```
[the remark this answers is not in the document]
Comment (yalsayyad) [reply false_positives/ap_pipe_HSC_RC2.md]: No, I didn't have any trouble with the repos. The butler and CmdlineTask were happy with the procedure.

File: false_positives/ap_pipe_HSC_RC2.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+from lsst.meas.algorithms import LoadIndexedReferenceObjectsTask
+
+config.differencer.refObjLoader.retarget(LoadIndexedReferenceObjectsTask)
+config.differencer.refObjLoader.load(os.path.join(getPackageDir("obs_subaru"), "config", "filterMap.py"))
+
+config.differencer.refObjLoader.ref_dataset_name = "ps1_pv3_3pi_20170110"
+config.ppdb.db_url="sqlite:////datasets/hsc/repo/rerun/private/yusra/RC2/ap_pipe_PDR1_bestcase_noSkyCorr/association.db"
+config.ppdb.isolation_level="READ_UNCOMMITTED"
+config.ppdb.connection_timeout=240
+config.differencer.doWriteMatchedExp=True
```
[the remark this answers is not in the document]
Comment (yalsayyad) [reply false_positives/ap_pipe_HSC_RC2.md]: You're right that the filterMap configs are a no-op if `doSelectSources` is False.  I'll clarify.

[the remark this answers is not in the document]
Comment (kfindeisen) [reply false_positives/ap_pipe_HSC_RC2.md]: Now I'm more confused. Is `doSelectSources` `False` from some other config?

[the remark this answers is not in the document]
Comment (yalsayyad) [reply false_positives/ap_pipe_HSC_RC2.md]: It's False by default in ap_pipe: https://github.com/lsst/ap_pipe/blob/master/python/lsst/ap/pipe/ap_pipe.py#L89

[the remark this answers is not in the document]
Comment (kfindeisen) [reply false_positives/ap_pipe_HSC_RC2.md]: D'oh! 🤦‍♂

**Comment by yalsayyad** [comment]
Didn't force push after rebasing master, so GitHub is not automatically registering the merge.

**Pull request closed without merging**